test/core/crypto/vrf/vrf_provider_test.cpp

```diff
@@ -59,16 +63,16 @@ TEST_F(VRFProviderTest, SignAndVerifySuccess) {
  */
 TEST_F(VRFProviderTest, VerifyFailed) {
   // given
-  VRFValue threshold{std::numeric_limits<VRFValue>::max() - 1};
+  VRFRawOutput threshold{std::numeric_limits<VRFValue>::max() - 1};
```

Comment:
Dont put it in test, but if we use 102084710076281554150585127412395147264 as threshold value, does the test pass?